arrow/arrow.py

```diff
@@ -736,8 +738,8 @@ def interval(
         representing a series of intervals between two inputs.
 
         :param frame: The timeframe.  Can be any ``datetime`` property (day, hour, minute...).
-        :param start: A datetime expression, the start of the range.
-        :param end: (optional) A datetime expression, the end of the range.
+        :param start: A datetime or Arrow expression, the start of the range.
+        :param end: (optional) A datetime or Arrow expression, the end of the range.
```

Comment:
please add a test case using the Arrow object - are we properly handling it in the body?